examples/openapi3/reverseproxy/app.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+'''
+example of connexion running behind a path-altering reverse-proxy
+
+NOTE this demo is not secure by default!!
```

Comment:
People will not read this and copy and paste the code. I would suggest to use `logging.warn` in the code as well to log this information. Whoever will do copy&paste will have more chances of seeing this.